k8s/nginx/staging/global-config.yaml

```diff
@@ -46,14 +46,14 @@ data:
       }
 
       internal;
-      resolver                kube-dns.kube-system.svc.cluster.local valid=5s;
-      proxy_method            $method;
```

Comment:
A some point, we shall have to revisit these spaces to understand if they actually matter. @Psalmz777 / @BenjaminSsempala